gems/pending/spec/spec_helper.rb

```diff
@@ -1,4 +1,6 @@
 require_relative '../bundler_setup'
+require 'azure-armrest'
```

Comment:
@jrafanie this goes back to the comment by @chrisarcand about the azure-armrest patch. Once that get's moved, this can be removed.